infrastructure/zk/src/run/run.ts

```diff
@@ -29,7 +29,8 @@ export async function deployERC20(command: 'dev' | 'new', name?: string, symbol?
     if (command == 'dev') {
         await utils.spawn(`yarn --silent --cwd contracts deploy-erc20 add-multi '[
             { "name": "wBTC", "symbol": "wBTC", "decimals":  8, "implementation": "RevertTransferERC20" },
-            { "name": "RDOC", "symbol": "RDOC", "decimals": 18 },
+            { "name": "RDOCs", "symbol": "RDOC", "decimals": 18 },
```

Comment:
Is the name changed to `RDOCs` deliberately, or as an accident?

Reply:
Probably a wrong `cmd+S`, fixed